server/datastore/mysql/scripts_test.go

```diff
@@ -356,3 +357,163 @@ func testListScripts(t *testing.T, ds *Datastore) {
 		})
 	}
 }
+
+func TestHostScripts(t *testing.T) {
+	ds := CreateMySQLDS(t)
```

Comment:
Any reason not to insert a sub-test here: https://github.com/fleetdm/fleet/pull/14199/files#diff-8d8a0f8241ea988443926569b7937de2b98a5d27d5e5abf9d52535d84cd70f87R25  and use `testGetHostScriptDetails(t *testing.T, ds *Datastore)` as function name? The benefit is that it reuses the same test databases (truncating tables between tests), making the suite a bit faster.